docs/installation.md

```diff
@@ -6,85 +6,168 @@ sidebar_position: 2
+by following the official documentation [here](https://filamentphp.com/docs/3.x/panels/installation).
+
+
+Once you setup your Laravel project with filament, you can install MoonGuard
+plugin via composer
 
 ```bash
-composer require taecontrol/moonguard
+composer require taecontrol/moonguard "^1.0.0@beta" -W
 ```
 
-## Migrations and Configuration file
+:::caution Heads Up
+Since MoonGuard is still in beta, set the `minimum-stability` in your 
+`composer.json` to `dev`
+:::
 
-To publish the migrations and the configuration file use:
+Publish the MoonGuard assets files and migrations file in your project.
+
+```php
+php artisan vendor:publish --tag="moonguard-assets"
 
-```bash
 php artisan vendor:publish --tag="moonguard-migrations"
-php artisan vendor:publish --tag="moonguard-config"
 ```
 
-You will be able to find a migration file called `create_moonguard_table.php` in your `database/migration` directory, and a configuration file called `config/moonguard.php`. To migrate the Moonguard table, run the following command:
+In production, run the migrations:
 
 ```bash
 php artisan migrate
 ```
 
-## Queues and Workers
+Go to `/moonguard` route to check the MoonGuard admin panel page:
 
-We use queues for notifications, so we recommend to configure Redis in your project and then run a worker:
+![login](./installation/login.png)
+
+If you don’t have any user, you can create a filament user to access to MoonGuard
+admin panel:
 
 ```bash
-php artisan queue:work
+php artisan make:filament-user
 ```
 
+ Login and you will land in the dashboard
+
+![dashboard](./installation/dashboard.png)
+
+## Moonguard Command Scheduler
+
+The **MoonGuard Command Scheduler** is a helper utility that executes all the
+Moonguard commands related to checks:
+
+- CheckUptimeCommand.
+- CheckSslCertificateCommand.
 
-## Development and local usage
+You can use this utility to set up MoonGuard task scheduling faster.
 
-In case you intend to extend or modify MoonGuard, you've to clone [MoonGuard repository](https://github.com/teacontrol/moonguard) in your disk.
+In order to use this utility, define a new schedule in your app and inside the
+`schedule` method of **`app/Console/Kernel.php`** . Use the static function
+`scheduleMoonGuardCommands()` from the **MCS** class, then pass the `$schedule`
+object and two cron strings (one for the Uptime Check and one for the SSL
+Certificate Check):
 
-Once you have clone it, update the `composer.json` of one of your projects where you intend to use MoonGuard as a dependecy, update the `repositories` key:
+```php
+<?php
 
-```json
+use Taecontrol\MoonGuard\Console\MoonGuardCommandsScheduler;
+//...
+
+protected function schedule(Schedule $schedule): void
 {
-  "repositories": [
-    {
-      "type": "path",
-      "url": "../<path>/moonguard"
-    }
-  ]
+  MoonGuardCommandsScheduler::scheduleMoonGuardCommands(
+    $schedule,
+    '* * * * *',
+    '* * * * *'
+  );
 }
 ```
 
```

Comment:
Here you can mention that MCS is scheduled running this command on the server:
`php artisan schedule:run`